discordClient.js

```diff
@@ -16,4 +14,21 @@ discordClient.on('ready', function (evt) {
     logger.info(discordClient.username + ' - (' + discordClient.id + ')');
 });
 
+var randomFunnyMessages = [
+    "I am back!",
+    "EROMBO",
+    "Moi Moi!",
```

Comment:
Moi Moi is goodbye, right? I think Moi (single) is what we want for Hello, unless this is being ironic.

Reply:
i just came up with random messages. but moi moi does mean goodbye